php/post-types/class-block-post.php

```diff
@@ -81,40 +81,57 @@ public function register_hooks() {
 	 * @return void
 	 */
 	public function register_controls() {
-		$controls = array(
-			'text'        => new Controls\Text(),
-			'textarea'    => new Controls\Textarea(),
-			'url'         => new Controls\URL(),
-			'email'       => new Controls\Email(),
-			'number'      => new Controls\Number(),
-			'color'       => new Controls\Color(),
-			'image'       => new Controls\Image(),
-			'select'      => new Controls\Select(),
-			'multiselect' => new Controls\Multiselect(),
-			'toggle'      => new Controls\Toggle(),
-			'range'       => new Controls\Range(),
-			'checkbox'    => new Controls\Checkbox(),
-			'radio'       => new Controls\Radio(),
+		$control_names = array(
+			'text',
+			'textarea',
+			'url',
```

Comment:
To make this work with `ucwords()`, the control class [is renamed](https://github.com/getblocklab/block-lab/pull/306/commits/addc1afeefcb5c777638bd484b5a1e476faa0242#diff-032d8fb82d07a7fc45dc4fd064a36ca8R3) from `Url` to `URL`.